pennylane/workflow/_capture_qnode.py

```diff
@@ -291,7 +290,50 @@ def _backprop(args, tangents, **impl_kwargs):
     return jax.jvp(partial(qnode_prim.impl, **impl_kwargs), args, tangents)
 
 
-diff_method_map = {"backprop": _backprop}
+def _finite_diff(args, tangents, **impl_kwargs):
+
+    gradient_kwargs = impl_kwargs["qnode_kwargs"]["gradient_kwargs"]
+    h = gradient_kwargs.get("h", 1e-6)
+    if gradient_kwargs.get("approx_order", 1) != 1:
+        raise NotImplementedError("only approx_order=1 is currently supported.")
+    if gradient_kwargs.get("strategy", "forward") != "forward":
+        raise NotImplementedError("only strategy='forward' is currently supported.")
+    available_kwargs = {"h", "strategy", "approx_order"}
```

Comment:
This might be overkill but is it worth considering standardizing what gradient kwarg belongs to which gradient transform? Do we ever expect these names to change?
```suggestion
    # this could be stored in gradient_transform.py
    FINITE_DIFF_SUPPORTED_KWARGS = {"h", "strategy", "approx_order"}
    available_kwargs = FINITE_DIFF_SUPPORTED_KWARGS
```